docs/src/examples/components/RadioGroup/Types/RadioGroupColorPickerExample.shorthand.tsx

```diff
@@ -22,37 +46,6 @@ class RadioGroupColorPickerExample extends React.Component {
       },
     }
   }
-
-  items = () => {
-    const colors = ['pink', 'blue', 'green', 'red', 'orange']
-    return colors.map(color => (
-      <RadioGroup.Item
-        value={color}
-        icon={this.createIcon(color)}
-        name={color}
-        key={color}
-        aria-label={color}
-      />
-    ))
-  }
-
-  state = { selectedValue: '' }
-  handleChange = (e, props) => {
-    this.setState({ selectedValue: props.value })
-  }
-  render() {
-    const { selectedValue } = this.state
-    return (
-      <div>
-        The selected value is: {selectedValue}
-        <Divider />
-        <RadioGroup
-          defaultCheckedValue="pink"
-          items={this.items()}
-          checkedValueChanged={this.handleChange}
-        />
-      </div>
-    )
-  }
 }
+
```

Comment:
This example should be definitely refactored in the future.

Reply:
if you have a good suggestion, I don't mind adding it now